build.sh

```diff
@@ -180,8 +178,8 @@ release() {
     git checkout -f $1
     git submodule update --init --checkout
     tag=$(git describe --tags --abbrev=0 ${1})
-
-    check_if_legacy $tag
+    check_version $tag
+    #check_if_legacy $tag
```

Comment:
@lwcislo why are we not checking for legacy? Do we really want to break legacy builds?